common/lib/permissions.js

```diff
@@ -25,6 +25,18 @@ const policePermissions = [
   'person_escort_record:confirm',
 ]
 
+const contractDeliveryManagerPermissions = [
+  'allocations:view',
+  'contract_delivery_manager',
```

Comment:
The pattern we've got for permissions is `area:permission` or `area:sub:permission` - would something like `locations:contract_delivery_manager` make any sense here? Just seeing `contract_delivery_manager` on its own looks weird compared to the other permissions